pkg/user/user_integration_test.go

```diff
@@ -65,13 +65,53 @@ func TestUserHandler(t *testing.T) {
 		var user2 model.User
 		client.PostJSON(t, "/users", strings.NewReader(`{
 			"email":    "user2@dhis2.org",
-			"password": "oneoneoneoneone1"
+			"password": "oneoneoneoneoneoneone111"
 		}`), &user2)
 
 		require.Equal(t, "user2@dhis2.org", user2.Email)
 		require.Empty(t, user2.Password)
 	}
 
+	t.Run("SignUpFailure", func(t *testing.T) {
```

Comment:
`SignUpFailed` would be more consistent with the naming of the existing test `SignInFailed`